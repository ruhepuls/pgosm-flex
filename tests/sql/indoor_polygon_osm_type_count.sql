SELECT osm_type COLLATE "C", COUNT(*)
    FROM osm.indoor_polygon
    GROUP BY osm_type COLLATE "C"
    ORDER BY osm_type COLLATE "C"
;