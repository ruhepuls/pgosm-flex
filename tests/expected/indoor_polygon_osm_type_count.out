corridor|36
no|1
room|262
wall|58
